builds/e2e/e2e.yaml

```diff
@@ -0,0 +1,87 @@
+# End-to-end tests should run after new docker images and edgelet packages are
+# built, but 'build completion triggers' are not yet supported in YAML. Set to
+# 'none' here and override in the online editor.
+trigger: none
+pr: none
+
+jobs:
+
+################################################################################	
+  - job: linux_arm32v7
+################################################################################	
+    displayName: Linux arm32v7
+
+    pool:
+      name: Azure-IoT-Edge-Core
+      demands: rpi3-e2e-tests
+
+    variables:
+      os: linux
```

Comment:
is it possible to use Agent.OS instead of creating a new variable "os"?

Reply:
Agent.OS won't work. On Linux, the value of Agent.OS is "Linux" with a capital "L", but I need lowercase for the docker image tag (probably a solvable problem, but not worth the effort). For Windows, I need "windows" and the variable returns "Windows_NT".